### PR TITLE
Adding timestamp to database backup

### PR DESCRIPTION
**Pull request by jaegeral**
Adressing: 

https://github.com/viper-framework/viper/issues/386

Tested it and the change should not break anything.

```
python update.py -d
[!] To update Projects you will need to copy their viper.db file in to the main viper folder
[!] Run the DB update then move the new db file back to the project folder. 
[*] Updating to New DB format
 - Backing up Sqlite DB
 - Creating New DataBase File
 - Connecting to Viper Databases
 - Reading data from Old Database
 -  Adding rows to New Database
[*] Update Complete
$ ls
binaries            viper.db
build               viper.py
data                viper_20160126-160238.db.bak
docs                web
[...]
```

### CODE REVIEW

**Comment by Rafiot** [comment]
@deralexxx Can you fix the conflict and merge your PR?

File: update.py [hunk 1]
```diff
@@ -103,7 +104,10 @@ def update_db():
     print_item("Backing up Sqlite DB")
 
     try:
-        os.rename('viper.db', 'viper.db.bak')
+    	# backup of database name with a timestamp to avoid to be overwritten
+    	# https://github.com/viper-framework/viper/issues/386
+    	db_backupname = "viper_{0}.db.bak".format(time.strftime("%Y%m%d-%H%M%S"))
```
Comment (Rafiot) [inline update.py]: tabs instead of spaces, can you also change it, please?

Comment (deralexxx) [inline update.py]: should be better now

**Comment by Rafiot** [comment]
@deralexxx it seems there is still a conflict. Do you mind having a look?

**Comment by deralexxx** [comment]
yeah will re-do it, guess I just have to re-do it, should not be a big deal

**Comment by Rafiot** [comment]
Great, thanks :)

**Pull request closed without merging**